cmd/out.go

```diff
@@ -53,7 +53,8 @@ var outCmd = &cobra.Command{
 		signal.Notify(c, os.Interrupt, syscall.SIGTERM)
 		go func() {
 			<-c
-			fmt.Println(" Interruption, saving last memory bits..")
+			fmt.Println("Interruption, saving last memory bits..")
+			rabbit.SafeStop()
```

Comment:
so this is about graceful shutdown on sigterm right? What's the consequence of not doing the SafeStop? Some stale channel on the broker?